### PR TITLE
[Keyboard] Add ginkgo65hot

### PR DESCRIPTION
**Pull request by Runheme**
Add ginkgo65hot

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mokey/ginkgo65hot/readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Ginkgo65hot Multi PCB
+
+![Ginkgo65hot](https://raw.githubusercontent.com/rhmokey/123.github.io/main/2021/08/15/hello-world/ginkgo65hot.png)
```
Comment (drashna) [inline keyboards/mokey/ginkgo65hot/readme.md:3]: Please rehost this image on imgur.

File: keyboards/mokey/ginkgo65hot/rules.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/mokey/ginkgo65hot/rules.mk:17]:
```suggestion
```

Comment (Runheme) [reply keyboards/mokey/ginkgo65hot/rules.mk:17]: Isn't that the same？

Comment (drashna) [reply keyboards/mokey/ginkgo65hot/rules.mk:17]: Nope. It is removing it.  The comments are deprecated, as is the sleep led feature.

Comment (Runheme) [reply keyboards/mokey/ginkgo65hot/rules.mk]: OK, they have been deleted

**Review by drashna — APPROVED** [review]

File: keyboards/mokey/ginkgo65hot/rules.mk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/mokey/ginkgo65hot/rules.mk:15]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by kemko** [comment]
@Runheme Sorry for irrelevant pinging in this issue, but I can't find any of our contacts. Can you publish qmk keymaps for ginkgo pro 65 hot anywhere?

**Pull request merged**